docs/blog/newsletter-april-2021.rst

```diff
@@ -0,0 +1,104 @@
+
+.. post:: April 01, 2021
+   :tags: newsletter
+
+######################################
+Write the Docs Newsletter – April 2021
+######################################
+
+Hello everybody! It's Beth and the team, here with the Write the Docs newsletter for April.
+
+Hopefully you'll be pleased to hear we've resisted any tempation to include an April Fool's prank. But I wouldn't want you to think that we're no fun, so here is `a delightful poem about the stuck ship in the Suez canal <https://twitter.com/krfabian/status/1374798847650709504>`__ by a friend of mine.
```

Comment:
This is extremely silly, and I'm pretty tempted to remove it - please tell me what you think.

Reply:
Doesn't do much for me.

Reply:
I like it! silly but timely.

Reply:
I think on balance I'll take it out.